example.json

```diff
@@ -13,6 +13,7 @@
         },
         "_questions": {
             "_resetType": "hard",
+            "_resetIncorrectOnly": false,
```

Comment:
Maybe rename this property to something beginning with 'can' or 'should' to indicate that it's a boolean and make consistent with other boolean properties below. E.g. `_canResetCorrectAnswers`